src/seq/minhash.jl

```diff
@@ -18,12 +18,12 @@ hashes for kmers of a given length. The type contains two parameters:
 """
 immutable MinHashSketch
     sketch::Vector{UInt64}
-    kmersize::Int
+    kmersize::Integer
```

Comment:
I think this should be kept as `Int`. Loosening the type of a field will propagate to other variables and result in performance degradation. In Julia, calling `new` automatically converts the types of arguments to those of fields. So, even if the constructor takes an `Integer` value, the value will be converted to `Int` when a sketch object is created.